electron/renderer.js

```diff
@@ -57,13 +58,25 @@ class Renderer {
         try {
             this.redirectOutputToConsole();
             mocha.setup({
-                ui: 'tdd'
+                ui: 'tdd',
             });
 
-            let mochaInst = new Mocha();
+            // Format the reporter options
+            let reporterOptions;
+
+            // Parse string as an object
+            if (typeof this.options.reporterOptions === "string") {
```

Comment:
I tried adding this after the `mocahInst` assignment, but it didn't take the options. The reporterOptions needs to be passed into the Mocha constructor because it's assigned to the reporter right away.

```js
 if (typeof this.options.reporterOptions === "string") {
    mochaInst.options.reporterOptions = querystring.parse(
        this.options.reporterOptions
    );
}
```